src/screens/Learning.js

```diff
@@ -113,12 +143,12 @@ export default ({
               button={
                 <LanguageSwitcher
                   firstLanguage={LANGUAGE_NAMES.EN}
-                  LeftText="EN"
-                  RightText="MA"
+                  LeftText={leftText}
+                  RightText={rightText}
                   color="#FFFFFF"
                   iconType=""
                   iconName="swap-horiz"
-                  onPress={() => null}
+                  onPress={() => toggleLanguageName()}
```

Comment:
should be able to pass just function without calling it
onPress={toggleLanguageName}